vite.config.ts

```diff
@@ -11,11 +11,16 @@ export default defineConfig({
   test: {
     globals: true,
     environment: "jsdom",
-    include: ["**/__tests__/**/*.ts", "**/__tests__/**/*.tsx"],
+    setupFiles: "src/tests/setup.ts",
+    include: ["**/__tests__/**/*.test.ts", "**/__tests__/**/*.test.tsx"],
     clearMocks: true,
     typecheck: {
       enabled: true,
     },
-    passWithNoTests: true,
+    css: {
+      modules: {
+        classNameStrategy: "non-scoped",
```

Comment:
For CSS Modules? How would it look?

Reply:
I want to not have stuff as "selected-26AHva" in the class. I want to test exact class.
This causes the classnames to not be scoped, so if something should get e.g. "selected" class, it would be "selected" in the test also